src/plugins/media/nodeviews/mediaNodeView/media.tsx

```diff
@@ -148,6 +175,40 @@ export class MediaNode extends Component<MediaNodeProps, MediaNodeState> {
     }
   }
 
+  fetchCozyImage = async () => {
+    if (!this.state.collabEditProvider)
+      throw Error('No CollabEdit was provided')
+
+    const {
+      serviceClient,
+      config: { noteId }
+    } = this.state.collabEditProvider
+
+    try {
+      const {
+        included
+      }: {
+        included: { id: string; links: { self: string } }[]
+      } = await serviceClient.cozyClient.query({
+        doctype: CozyDoctypes.Files,
+        id: noteId
+      })
```

Comment:
you can use `Q`: 

client.query(Q(CozyDoctypes.Files).getById(noteId))`

Reply:
Updated